README.md

```diff
@@ -1,12 +1,101 @@
 # Genoa
 WIP: Opinionated webhook based GitOps controller that installs and manages helm (v3) releases from configured github/gitlab repositories
 
+Genoa does two important things:
+
+* Allows you to manage your helm releases using a custom resource definitions of the chart name, version, and values
+* Responds to git webhook invocations when those resource definitions change (gitops)
+
+![image info](./pictures/genoa.png)
+
 ## Installing the Chart
 You can install the chart with the release name `genoa` as below.
 
+From this repository, create your own `genoa.values.yaml` file and install the chart:
+
 ```console
 $ kubectl create ns genoa
-$ helm upgrade genoa charts/genoa --install --namespace=genoa
+$ helm upgrade genoa charts/genoa --install --namespace=genoa --values=genoa.values.yaml
 ```
+NOTE: For custom helm repositories, you need to tell Genoa about them with the `config.helmRepos` in your Genoa values.yaml file.
+The  can be viewed in `repositories.yaml` file in this repository.
```

Comment:
"The sample repository config structure can be viewed... "
or 
chart values file: `.Values.config.helmRepos`